packages/commons/src/authentication.ts

```diff
@@ -28,58 +31,151 @@ export const userScopes = {
   config: 'config'
 } as const
 
-export const userRoleScopes = {
-  FIELD_AGENT: [userScopes.declare],
-  REGISTRATION_AGENT: [
-    userScopes.validate,
-    userScopes.performance,
-    userScopes.certify
-  ],
-  LOCAL_REGISTRAR: [
-    userScopes.register,
-    userScopes.performance,
-    userScopes.certify
-  ],
-  LOCAL_SYSTEM_ADMIN: [userScopes.systemAdmin],
-  NATIONAL_SYSTEM_ADMIN: [
-    userScopes.systemAdmin,
-    userScopes.nationalSystemAdmin
-  ],
-  PERFORMANCE_MANAGEMENT: [userScopes.performance],
-  NATIONAL_REGISTRAR: [
-    userScopes.register,
-    userScopes.performance,
-    userScopes.certify,
-    userScopes.config,
-    userScopes.teams
-  ]
-}
-
 /** All the scopes system/integration can be assigned to */
-export const systemScopes = {
+export const SYSTEM_INTEGRATION_SCOPES = {
   recordsearch: 'recordsearch',
   declare: 'declare',
-  notificationApi: 'notification-api',
-  validatorApi: 'validator-api',
-  ageVerificationApi: 'age-verification-api',
   webhook: 'webhook',
   nationalId: 'nationalId'
 } as const
 
-export const systemRoleScopes = {
-  HEALTH: [systemScopes.declare, systemScopes.notificationApi],
-  NATIONAL_ID: [systemScopes.nationalId],
-  EXTERNAL_VALIDATION: [systemScopes.validatorApi],
-  AGE_CHECK: [systemScopes.declare, systemScopes.ageVerificationApi],
-  RECORD_SEARCH: [systemScopes.recordsearch],
-  WEBHOOK: [systemScopes.webhook]
-}
+export const DEFAULT_CORE_ROLE_SCOPES = {
+  FIELD_AGENT: ['declare'],
+  REGISTRATION_AGENT: ['validate', 'performance', 'certify'],
+  LOCAL_REGISTRAR: [
+    'register',
+    'performance',
+    'certify',
+
+    'record.register',
+    'record.declaration-review',
+    'record.print-issue-certified-copies'
+  ],
+  LOCAL_SYSTEM_ADMIN: ['sysadmin'],
+  NATIONAL_SYSTEM_ADMIN: ['sysadmin', 'natlsysadmin'],
+  PERFORMANCE_MANAGEMENT: ['performance'],
+  NATIONAL_REGISTRAR: ['register', 'performance', 'certify', 'config', 'teams'],
+  SUPER_ADMIN: ['natlsysadmin', 'bypassratelimit', 'sysadmin']
+} satisfies Record<string, Scope[]>
+
+// @TODO: What is the point of this, should we use this in `packages/auth/src/features/scopes/service.ts`?
+export const DEFAULT_ROLES_DEFINITION = [
+  {
+    id: 'FIELD_AGENT',
```

Comment:
Like Riku already explained, this and DEFAULT_CORE_ROLE_SCOPES should be combined. We should only need DEFAULT_ROLES_DEFINITION.